doc/source/operations/upgrading-openstack.rst

```diff
@@ -246,6 +246,9 @@ suggestions:
 * Check Grafana dashboards.
 * Update the deployment to use the latest |previous_release| images and
   configuration.
+* If your customer has overriden any policies, check to see if they need
+  updating to align with new defaults. These will be written to files
+  ``kolla/config/<service>/policy.yaml``.
```

Comment:
Could we include a note on how someone would learn what the new defaults are?

Reply:
Policy reference documentation can generally be found in the documentation of each project. For example, Nova policy: https://docs.openstack.org/nova/latest/configuration/policy.html

Reply:
> Policy reference documentation can generally be found in the documentation of each project. For example, Nova policy: https://docs.openstack.org/nova/latest/configuration/policy.html

@MoteHue could you add this line to the docs?

Reply:
Shamelessly stole Pierre's wording, thanks :)